2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 15 Line 1_
```python
string_to_index = 'this is a string that will be indexed'
```
One thing I really like about the r4wrds course is that all the examples use watery/geo/DS-ey examples.

Like `flow_rate_gpm = 42`, or addresses / streamgage names as strings.

It's kinda gimicky, but it's also a way we can add value vs off-the-shelf / open-source courses.

I don't think it's worth changing here, but worth doing in future modules I reckon.